behat/features/bootstrap/FeatureContext.php

```diff
@@ -67,4 +67,133 @@ public function iVisitNodePageOfType($title, $type) {
   public function iShouldHaveAccessToThePage() {
     $this->assertSession()->statusCodeEquals('200');
   }
+
+  /**
+   * @Then I should not have access to the page
+   */
+  public function iShouldNotHaveAccessToThePage() {
+    $this->assertSession()->statusCodeEquals('403');
+  }
+
+  /**
+   * @When I start editing :title node of type :type
+   */
+  public function iStartEditingNodeOfType($title, $type) {
+    $query = new \entityFieldQuery();
+    $result = $query
+      ->entityCondition('entity_type', 'node')
+      ->entityCondition('bundle', strtolower($type))
+      ->propertyCondition('title', $title)
+      ->propertyCondition('status', NODE_PUBLISHED)
+      ->range(0, 1)
+      ->execute();
+    if (empty($result['node'])) {
+      $params = array(
+        '@title' => $title,
+        '@type' => $type,
+      );
+      throw new \Exception(format_string("Node @title of @type not found.", $params));
+    }
+    $nid = key($result['node']);
+    $this->getSession()->visit($this->locatePath('node/' . $nid . '/edit'));
+  }
+
+  /**
+   * @When I start creating node of type :type
+   */
+  public function iStartCreatingNodeOfType($type) {
+    $this->getSession()->visit($this->locatePath('node/add/' . $type));
+  }
+
+  /**
+   * @When I create :title node of type :type
+   */
+  public function iCreateNodeOfType($title, $type) {
+    $account = user_load_by_name($this->user->name);
+    $values = array(
+      'type' => $type,
+      'uid' => $account->uid,
+      'status' => 1,
+      'comment' => 1,
+      'promote' => 0,
+    );
+    $entity = entity_create('node', $values);
+    $wrapper = entity_metadata_wrapper('node', $entity);
+    $wrapper->title->set($title);
```

Comment:
Can't use ``$wrapper->label->set()``, getting an error. As I understand, for entity metadata wrapper there is only ``label()`` that returns label or title of the entity. But to set the value we should use the ``title`` name of the field.